modules/ROOT/pages/amazon/amazon-dynamodb-connector-reference.adoc

```diff
@@ -1,1190 +1,1167 @@
+| Socket Timeout a| Number |  The amount of time to wait (in milliseconds) for data to be transferred over an established, open connection before the connection times out. A value of 0 means infinity and is not recommended. |  50000 | 
+| Access Key a| String |  The access key provided by Amazon. |  | x
+| Secret Key a| String |  The secret key provided by Amazon. |  | x
+| Try Default AWSCredentials Provider Chain a| Boolean |  Set this field to true to obtain credentials from the AWS environment. See the Amazon https://docs.aws.amazon.com/sdk-for-java/v2/developer-guide/credentials.html[credentials] topic. |  false | 
+| Max Connections a| Number |  Sets the maximum number of allowed open HTTP connections. |  | 
+| Region Endpoint a| String |  Set the topic region endpoint. |  US_EAST_1 | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  | 
+| Host a| String |  The optional proxy host. |  | 
+| Port a| Number |  The optional proxy port. |  | 
+| Username a| String |  The optional proxy username. |  | 
+| Password a| String |  The optional proxy password. |  | 
+| Domain a| String |  The optional proxy domain. |  | 
+| Workstation a| String |  The optional proxy workstation. |  | 
+|===
+[[config_role]]
+===== Role
+
+
+====== Parameters
+[%header,cols="20s,20a,30a,20a,10a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Connection Timeout a| Number |  |  50000 |
-| Socket Timeout a| Number |  |  50000 |
-| Access Key a| String |  |  |x
-| Secret Key a| String |  |  |x
-| Try Default AWSCredentials Provider Chain a| Boolean |  |  false |
-| Max Connections a| Number |  |  |
-| Region Endpoint a| Enumeration, one of:
-
-** AP_NORTHEAST_1
-** AP_NORTHEAST_2
-** AP_SOUTH_1
-** AP_SOUTHEAST_1
-** AP_SOUTHEAST_2
-** CA_CENTRAL_1
-** CN_NORTH_1
-** EU_CENTRAL_1
-** EU_WEST_1
-** EU_WEST_2
-** SA_EAST_1
-** US_EAST_1
-** US_EAST_2
-** US_GOV_WEST_1
-** US_WEST_1
-** US_WEST_2
-|  |  US_EAST_1 |
-| Reconnection a| Reconnection |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  |
-| Host a| String |  |  |
-| Port a| Number |  |  |
-| Username a| String |  |  |
-| Password a| String |  |  |
-| Domain a| String |  |  |
-| Workstation a| String |  |  |
-|===
-
-==== Associated Operations
-
-* batchDeleteItem
-* batchGetItem
-* batchPutItem
-* createTable
-* deleteItem
-* deleteTable
-* describeTable
-* getItem
-* listTables
-* putItem
-* query
-* scan
-* updateItem
-* updateTable
+| Role ARN a| String |  The Role ARN unique identifies role to assume to gain cross account access. |  | x
```

Comment:
This needs rewording. I suggest this wording, if it's correct: 

"The Role ARN uniquely identifies the role used to gain cross-account access."